env0/resource_policy.go

```diff
@@ -0,0 +1,182 @@
+package env0
+
+import (
+	"context"
+	"errors"
+	"log"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/google/uuid"
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourcePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePolicyCreate,
+		ReadContext:   resourcePolicyRead,
+		UpdateContext: resourcePolicyUpdate,
+		DeleteContext: resourcePolicyReset,
+
+		Importer: &schema.ResourceImporter{StateContext: resourcePolicyImport},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Description: "id of the policy",
+				Computed:    true,
+			},
+			"project_id": {
+				Type:        schema.TypeString,
+				Description: "id  of the project",
+				Required:    true,
+				ValidateDiagFunc: func(i interface{}, path cty.Path) diag.Diagnostics {
+					projectId := i.(string)
+					if projectId == "" {
+						return diag.Errorf("Project ID cannot be empty")
+					}
+					return nil
+				},
```

Comment:
This is redundant. Having the `project_id` attribute schema to include the `Required` setting is enough...